examples/buzzer/main.cc

```diff
@@ -25,25 +25,48 @@
 
 using Note = bsp::BuzzerNote;
 
-static bsp::BuzzerNoteDelayed startup[] = {
-  { Note::Mi3M,   80 },
-  { Note::Silent, 80 }, 
-  { Note::Mi3M,   80 },
-  { Note::Silent, 240 },
-  { Note::Mi3M,   80 },
-  { Note::Silent, 240 },
-  { Note::Do1M,   80 },
-  { Note::Silent, 80 },
-  { Note::Mi3M,   80 },
-  { Note::Silent, 240 },
-  { Note::So5M,   80 },
-  { Note::Silent, 560 },
-  { Note::So5L,   80  },
-  { Note::Silent, 560 },
-  { Note::Finish, 0 },
+static bsp::BuzzerNoteDelayed Mario[] = {
+    {Note::Mi3M, 80},    {Note::Silent, 80}, {Note::Mi3M, 80},
+    {Note::Silent, 240}, {Note::Mi3M, 80},   {Note::Silent, 240},
+    {Note::Do1M, 80},    {Note::Silent, 80}, {Note::Mi3M, 80},
+    {Note::Silent, 240}, {Note::So5M, 80},   {Note::Silent, 560},
+    {Note::So5L, 80},
+};
```

Comment:
I use Clang-Format to format my code into Google style, so you may see that there are some pieces of code formatted automatically by Clang-Format into Google style.

Reply:
That is okay for now, but try not to format things as we have not yet come to an agreement on which format we should follow yet.

Reply:
Ok. I already tested the code. It functions well. But I will not format any code next time.